.gitmodules

```diff
@@ -16,3 +16,7 @@
 [submodule "lib/htsget/htsget_app"]
 	path = lib/htsget/htsget_app
 	url = https://github.com/CanDIG/htsget_app.git
+[submodule "lib/candig-ingest/candigv2-ingest"]
+	path = lib/candig-ingest/candigv2-ingest
+	url = https://github.com/CanDIG/candigv2-ingest
+	branch = develop
```

Comment:
Let's keep the submodules file consistent and not include branch names (we would then need to keep separate versions for dev and prod, and change the instructions for how we update submodules) .